### DISCUSSION TITLE
Why OrchardCore.Contents.TagHelpers is located into its own assembly

### DISCUSSION THREAD

**Discussion by hishamco** [OrchardCMS/OrchardCore#9253] [Q&A]
I'm just wondering why `OrchardCore.Contents.TagHelpers` has it's own assembly and not in `OrchardCore.Content.Core` assembly?

**Comment by deanmarcussen** [comment]
Please refer notes on the pr https://github.com/OrchardCMS/OrchardCore/pull/5258

**Reply by hishamco** [reply]
This mean we will do the same thing for other projects
